src/mocks/mail.php

```diff
@@ -22,6 +22,8 @@ class Mock_Mailer extends \PHPMailer {
 	}
 } else {
 	class Mock_Mailer {
+		public function __construct() {
```

Comment:
Should also fix the call to `parent::__construct` here: https://github.com/penske-media-corp/pmc-unit-test/blob/main/src/traits/mocker.php#L23 to check for if the `parent::__construct` is defined and accessible since constructor can have a private modifier.